examples.manifest.yaml

```diff
@@ -50,3 +50,15 @@
   addr:
     host: localhost
     port: 9009
+- name: ilp-from-conf
```

Comment:
is there any value in having this example given all examples were changes to use a conf string?

Reply:
Yes: This example here is so that we can auto-generate an example from the cloud,
substituted with just the right config string.